Models/Crime.cs

```diff
@@ -2,16 +2,8 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
-// <auto-generated />
-//
-// To parse this JSON data, add NuGet 'Newtonsoft.Json' then do:
-//
-//    using QuickType;
-//
-//    var neighbourhoodAlert = NeighbourhoodAlert.FromJson(jsonString);
-namespace Neighbourhood_Alert
-
 
+namespace NeighbourhoodCrime
```

Comment:
Both of these changes look good: remove commented code, and remove the _ from the class name.